application/controllers/collection_instrument.py

```diff
@@ -453,6 +462,15 @@ def get_instrument_data(instrument_id, session):
 
         data = None
         file_name = None
+
```

Comment:
I think it might be better to query the database result for a file location and if that exists, then go get the file from the bucket and if it doesn't exist then load the data from the database as it currently does. 

At the minute you're trying to retrieve everything from the bucket, then catching and logging the error and if that fails then getting the db result. 

At first most CIs will be retrieved from the database so this will result in a lot of errors in our logs, when they're not really errors.